src/python/grpcio_tests/tests_aio/unit/channel_test.py

```diff
@@ -52,6 +55,26 @@ def test_unary_unary(self):
 
         self.loop.run_until_complete(coro())
 
+    def test_unary_call_times_out(self):
```

Comment:
It would be helpful to add a sunny day test case, just specify a timeout, and make sure it doesn't break the normal RPC.